modules/otel-collector-components.adoc

```diff
@@ -303,13 +393,13 @@ The Memory Limiter processor periodically checks the Collector's memory usage an
 |===
 
 [id="resource-detection-processor_{context}"]
-=== Resource Detection processor
+=== Resource Detection Processor
 
-The Resource Detection processor is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
+The Resource Detection Processor is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
 
-The Resource Detection processor identifies host resource details in alignment with OpenTelemetry's resource semantic standards. Using the detected information, it can add or replace the resource values in telemetry data. This processor supports traces, metrics, and can be used with multiple detectors such as the Docket metadata detector or the `OTEL_RESOURCE_ATTRIBUTES` environment variable detector.
+The Resource Detection Processor identifies host resource details in alignment with OpenTelemetry's resource semantic standards. Using the detected information, it can add or replace the resource values in telemetry data. This processor supports traces, metrics, and can be used with multiple detectors such as the Docket metadata detector or the `OTEL_RESOURCE_ATTRIBUTES` environment variable detector.
```

Comment:
```suggestion
The Resource Detection Processor identifies host resource details in alignment with OpenTelemetry's resource semantic standards. By using the detected information, it can add or replace the resource values in telemetry data. This processor supports traces, metrics, and you can use it with multiple detectors such as the Docket metadata detector or the `OTEL_RESOURCE_ATTRIBUTES` environment variable detector.
```

Reply:
Good to have (if you did care to fix it)

Reply:
Thank you for the suggestion. I didn't add "By" to "Using the detected information" because the "detected information" is not a tool that performs the action (compared to say "By using the processor").